BenchmarkDotNet fail in WPF project with .NET 5.0-windows target

Step:

1. Create a WPF project
2. Add BenchmarkDotNet package
3. Add and run any Benchmark code

Running the Benchmark code that we can find error NU1201: Project Benchmark is not compatible with net5.0 (.NETCoreApp,Version=v5.0). Project Benchmark supports: net5.0-windows7.0 (.NETCoreApp,Version=v5.0)

```
 Standard error:
   Restored f:\Code\lindexi\WpfInk\Benchmark\Benchmark.csproj (in 275 ms).
  Restored f:\Code\lindexi\WpfInk\WpfInk Old\WpfInkOld.csproj (in 84 ms).
  Restored f:\Code\lindexi\WpfInk\WpfInk\WpfInk.csproj (in 84 ms).
  Failed to restore f:\Code\lindexi\WpfInk\Benchmark\bin\Release\net5.0-windows\82790c3e-0ce6-4852-a0db-f9674aea9be3\BenchmarkDotNet.Autogenerated.csproj (in 86 ms).
f:\Code\lindexi\WpfInk\Benchmark\bin\Release\net5.0-windows\82790c3e-0ce6-4852-a0db-f9674aea9be3\BenchmarkDotNet.Autogenerated.csproj : error NU1201: Project Benchmark is not compatible with net5.0 (.NETCoreApp,Version=v5.0). Project Benchmark supports: net5.0-windows7.0 (.NETCoreApp,Version=v5.0)
  Determining projects to restore...

// Build Error: Standard output:

 Standard error:
   Restored f:\Code\lindexi\WpfInk\Benchmark\Benchmark.csproj (in 275 ms).
  Restored f:\Code\lindexi\WpfInk\WpfInk Old\WpfInkOld.csproj (in 84 ms).
  Restored f:\Code\lindexi\WpfInk\WpfInk\WpfInk.csproj (in 84 ms).
  Failed to restore f:\Code\lindexi\WpfInk\Benchmark\bin\Release\net5.0-windows\82790c3e-0ce6-4852-a0db-f9674aea9be3\BenchmarkDotNet.Autogenerated.csproj (in 86 ms).
f:\Code\lindexi\WpfInk\Benchmark\bin\Release\net5.0-windows\82790c3e-0ce6-4852-a0db-f9674aea9be3\BenchmarkDotNet.Autogenerated.csproj : error NU1201: Project Benchmark is not compatible with net5.0 (.NETCoreApp,Version=v5.0). Project Benchmark supports: net5.0-windows7.0 (.NETCoreApp,Version=v5.0)
  Determining projects to restore...
```

Demo: https://github.com/lindexi/lindexi_gd/tree/f0b45eecb63e6ceef7a3abe5b43503abb317fd70/WpfInk

--- a/src/BenchmarkDotNet/Environments/Runtimes/CoreRuntime.cs
+++ b/src/BenchmarkDotNet/Environments/Runtimes/CoreRuntime.cs
@@ -208,9 +208,7 @@ namespace BenchmarkDotNet.Environments
             if (!(platformNameProperty.GetValue(attributeInstance) is string platformName))
                 return fallback;
 
-            // it's something like "Windows7.0";
-            var justName = new string(platformName.TakeWhile(char.IsLetter).ToArray());
-            return new CoreRuntime(fallback.RuntimeMoniker, $"{fallback.MsBuildMoniker}-{justName}", fallback.Name);
+            return new CoreRuntime(fallback.RuntimeMoniker, $"{fallback.MsBuildMoniker}-{platformName}", fallback.Name);
         }
     }
 }

